backend/server.ts

```diff
@@ -80,6 +86,7 @@ app.post('/login', async (req, res) => {
 import enemyRoutes from './routes/enemy'
 import playerRoutes from './routes/player'
```

Comment:
is there a reason why these import statements are in the middle and not with the rest of the import statements at the top?